ang/civicase/Popover.html

```diff
@@ -0,0 +1,10 @@
+<div class="civicase__popover"
+  aria-haspopup="true"
+  aria-expanded="{{isOpen}}">
+  <span ng-transclude="toggleButton"></span>
+  <div class="popover bottom civicase__popover-box {{popoverClass}}" ng-if="isOpen">
```

Comment:
Can we implement `class="popover bottom civicase__popover-box {{popoverClass}}"`  using `ng-class` ?